asyncsleepiq/api.py

```diff
@@ -174,9 +174,9 @@ async def login_cookie(self, email: str, password: str) -> None:
 
     async def put(
         self, url: str, json: dict[str, Any] = {}, params: dict[str, Any] = {}
-    ) -> None:
+    ) -> dict[str, Any]:

```

Comment:
Since the `bamkey` function needs the response data, I added a return type and return statement to this function

https://github.com/kbickar/asyncsleepiq/pull/13/files#diff-017b6dfa53f4adfdbca910dbe1e8e8abb330d0dc58b6f8491b58c7e40dc4257fL204